lib/ansible/modules/packaging/os/package_facts.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/python
+# (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# most of it copied from AWX's scan_packages module
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: package_facts
+short_description: package information as facts
+description:
+  - Return information about installed packages as facts
+options:
+  manager:
+    description:
+      - The package manager used by the system so we can query the package information
+    default: auto
+    choices: ["auto", "rpm", "zypper", "dnf", "yum", "dpkg", "apt", "aptitude"]
```

Comment:
this seems misleading, really only rpm and deb are the choices

Reply:
@bcoca - fair point, I'll update. I was originally thinking the alias idea made sense because they are categorically front ends but it is ultimately misleading and that's not ideal.